perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -133,6 +133,9 @@ sub send_alert_type {
                 $row->{item_text_original} = $row->{item_text};
                 $row->{item_text} = $row->{item_text} ? $row->{item_text} . "\n\n" . $update :
                                                         $update;
+                if ($row->{item_private_email_text} && $report->contact->get_extra_metadata('type', '') ne 'waste') {
```

Comment:
Just curious if this could be `$report->cobrand_data ne 'waste'` instead?

Reply:
Aha yes, appears so